web/modules/custom/contact_information/contact_information.module

```diff
@@ -22,17 +22,15 @@ function contact_information_entity_bundle_field_info_alter(&$fields, EntityType
  *
  * Concatenates the names into a single string. This entity has support for
  * multiple names and entity label does not support this. For this reason,
- * the label actually inherits all values from field_ci_name field.
+ * the label actually concatenates all values from field_ci_name field.
  */
```

Comment:
I would rather move this documentation in the auto_label function, and simplify this one.